docs/integrations/security-threat-detection/threat-intel-quick-analysis.md

```diff
@@ -39,11 +39,11 @@ The Threat Intel Quick Analysis App provides baseline queries. You can further o
 For example:
 
 ```sql
-_sourceCategory=cylance "IP Address"
-| parse regex "(?<ip_address>\b\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})"
+_sourceCategory=cylance ""IP Address""
+| parse regex ""(?<ip_address>\b\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})""
 | where !isNull(ip_address)
-| where ip_address != "0.0.0.0" and ip_address != "127.0.0.1"
-| lookup type, actor, raw, threatlevel as malicious_confidence from sumo://threat/cs on threat=ip_address
+| where ip_address != ""0.0.0.0"" and ip_address != ""127.0.0.1""
```

Comment:
Same question as above. Why are there two sets of quotation marks?

Reply:
Updated these queries which has two double quotes. Thanks for identifying this John